This patch is applied by update-opentelemetry-proto.sh to the otel protos.

diff --git a/pkg/obsservice/obspb/opentelemetry-proto/logs/v1/logs.proto b/pkg/obsservice/obspb/opentelemetry-proto/logs/v1/logs.proto
index 9629e39e9c..d5db90161f 100644
--- a/pkg/obsservice/obspb/opentelemetry-proto/logs/v1/logs.proto
+++ b/pkg/obsservice/obspb/opentelemetry-proto/logs/v1/logs.proto
@@ -19,6 +19,8 @@ package opentelemetry.proto.logs.v1;
 import "obsservice/obspb/opentelemetry-proto/common/v1/common.proto";
 import "obsservice/obspb/opentelemetry-proto/resource/v1/resource.proto";
 
+import "gogoproto/gogo.proto";
+
 option csharp_namespace = "OpenTelemetry.Proto.Logs.V1";
 option java_multiple_files = true;
 option java_package = "io.opentelemetry.proto.logs.v1";
@@ -53,7 +55,7 @@ message ResourceLogs {
   opentelemetry.proto.resource.v1.Resource resource = 1;
 
   // A list of ScopeLogs that originate from a resource.
-  repeated ScopeLogs scope_logs = 2;
+  repeated ScopeLogs scope_logs = 2 [(gogoproto.nullable) = false ];
 
   // This schema_url applies to the data in the "resource" field. It does not apply
   // to the data in the "scope_logs" field which have their own schema_url field.
@@ -68,7 +70,7 @@ message ScopeLogs {
   opentelemetry.proto.common.v1.InstrumentationScope scope = 1;
 
   // A list of log records.
-  repeated LogRecord log_records = 2;
+  repeated LogRecord log_records = 2 [(gogoproto.nullable) = false ];
 
   // This schema_url applies to all logs in the "logs" field.
   string schema_url = 3;
